tools/createValidate.js

```diff
@@ -0,0 +1,132 @@
+const fs = require("fs");
```

Comment:
Can you add a file description explaining how to use this tool and it's purpose briefly?